### ISSUE TITLE
Tarball for pui-css-collapse 7.5.0 missing collapse.css

### ISSUE THREAD

**Issue by pivotal-todd-ritchie** [pivotal-cf/pivotal-ui#474]
We're getting `ENOENT` on `collapse.css` when trying to build assets. It looks like `collapse.css` disappeared from the package.

```
$ gulp assets
...
Error: ENOENT: no such file or directory, open '/Users/pivotal/workspace/toolsmiths-envs/node_modules/pui-css-collapse/collapse.css'
    at Error (native)
```

Here's a diff of the files provided by the tarballs:

```
$ tar vft pui-css-collapse-7.4.0.tgz
-rw-r--r--  0 501    20        688 Jan 18 08:21 package/package.json
-rw-r--r--  0 501    20       1148 Jan 18 08:21 package/README.md
-rw-r--r--  0 501    20       1089 Jan 18 08:21 package/LICENSE
-rw-r--r--  0 501    20        147 Jan 18 08:21 package/index.js
-rw-r--r--  0 501    20       1227 Jan 18 08:21 package/collapse.css

$ tar vft pui-css-collapse-7.5.0.tgz
-rw-r--r--  0 501    20        688 Feb  3 09:44 package/package.json
-rw-r--r--  0 501    20       1148 Feb  3 09:44 package/README.md
-rw-r--r--  0 501    20       1089 Feb  3 09:44 package/LICENSE
-rw-r--r--  0 501    20        147 Feb  3 09:44 package/index.js
```

**Comment by rdy** [comment]
I'm seeing this issue on some other stuff as well, can we potentially yank 7.5.0 or publish a version 7.5.1 as well?

**Comment by charleshansen** [comment]
Super weird, it looks like the `collapse.css` file somehow didn't make it to the npm registry. No idea how. We can publish a new version.

**Comment by charleshansen** [comment]
We published a 7.5.1 which appears to have the file

**Comment by jadekler** [comment]
Fixed in 7.5.1
